spec/index.md

```diff
@@ -622,7 +622,7 @@ In order to read parts of the [The system state tree](#state-tree), the user mak
 
 -   `sender`, `nonce`, `ingress_expiry`: See [Authentication](#authentication)
 
--   `paths` (sequence of paths): A list of paths, where a path is itself a sequence of blobs.
+-   `paths` (sequence of paths): A list of at most 1000 paths, where a path is itself a sequence of blobs of maximum length 127 blobs.
```

Comment:
For the second part of the sentence, how about something like "where a path is itself a sequence of at most 127 blobs"? In any case, one of the two occurrences of "blobs" should be removed.

Reply:
Good suggestion! Implemented.